Python/optimizer.c

```diff
@@ -373,6 +374,7 @@ translate_bytecode_to_trace(
     _PyUOpInstruction *trace,
     int max_length)
 {
+    _Py_CODEUNIT *initial_instr = instr;
```

Comment:
This generates a warning, so you'll need to guard it with `Py_DEBUG`

Reply:
Hah, that’s because I have plans for this variable in EXTENDED_ARG handling. But that’s for the future. So I will add the guard and merge the PR.